src/main/java/com/exactpro/th2/check1/event/VerificationEntryUtils.java

```diff
@@ -55,6 +56,9 @@ public static VerificationEntry createVerificationEntry(ComparisonResult result)
 
     private static String resolveOperation(ComparisonResult result) {
         Object expected = result.getExpected();
+        if (expected instanceof IOperationFilter) {
+            return ((IOperationFilter)expected).getOperation().name();
+        }
```

Comment:
We should move this statement into the `if (expected instanceof IFilter)` block because IOperationFilter extends IFilter